qt/widgets/common/src/FitPropertyBrowser.cpp

```diff
@@ -3277,9 +3214,9 @@ void FitPropertyBrowser::addAllowedSpectra(const QString &wsName, const QList<in
   auto ws = AnalysisDataService::Instance().retrieveWS<MatrixWorkspace>(name);
   if (ws) {
     QList<int> indices;
-    for (auto const i : wsSpectra) {
-      indices.push_back(static_cast<int>(ws->getIndexFromSpectrumNumber(i)));
-    }
+    std::transform(wsSpectra.begin(), wsSpectra.end(), std::back_inserter(indices),
```

Comment:
Same question here with using constant iterators and reserving memory? And also is it possible for the `int` in the lambda to be a `const int`?

Reply:
Done, as previous.